pkg/build/controller/strategy/custom.go

```diff
@@ -113,6 +113,7 @@ func (bs *CustomBuildStrategy) CreateBuildPod(build *buildapi.Build) (*kapi.Pod,
 		setupDockerSocket(pod)
 		setupDockerSecrets(pod, build.Spec.Output.PushSecret, strategy.PullSecret, build.Spec.Source.Images)
 	}
+	setOwnerReference(pod, build)
```

Comment:
there is quite a lot of overlap between the implemention of these three strategies - maybe a follow-up issue?